VariantShredding.md

```diff
@@ -42,7 +42,31 @@ Variant values are stored in Parquet fields named `value`.
 Each `value` field may have an associated shredded field named `typed_value` that stores the value when it matches a specific type.
 When `typed_value` is present, readers **must** reconstruct shredded values according to this specification.
 
-For example, a Variant field, `measurement` may be shredded as long values by adding `typed_value` with type `int64`:
+The Parquet columns used to store variant metadata and values must be accessed by name, not by position.
+
+In order to avoid ambiguity, this specification always uses the term "`null`" to mean the variant
+null value (binary encoding: `00`). The phrase "missing" or "NULL" (all caps) always refers to an
```

Comment:
In the spec, "missing" is used to refer to an object field that is not present in the object. For example, the field "type" is missing in this object: `{"id": 34}`. I don't think it is correct to conflate missing with a SQL null. Missing means the field is not present in an object.